feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
```

Comment:
This is good that you have added this line.

Reply:
Thanks for this comment.